server/integration/testsuite/src/test/java/org/infinispan/server/test/category/HotRodLocalDomain.java

```diff
@@ -0,0 +1,13 @@
+package org.infinispan.server.test.category;
+
+import org.junit.experimental.categories.Category;
+
+/**
+ * {@link Category} tag for local hot rod tests in domain mode.
+ * 
+ * @author mgencur
+ * 
+ */
+public class HotRodLocalDomain extends ClientLocal {
```

Comment:
Maybe HotRodSingleNodeDomain would be better name? It's a test with a single ISPN server (vs. Clustered where there are two servers)